azurelinuxagent/common/protocol/restapi.py

```diff
@@ -68,9 +72,21 @@ class VMAgentManifest(object):
     def __init__(self, family, version=None):
         self.family = family
         # This is the Requested version as specified by the Goal State, it defaults to 0.0.0.0 if not specified in GS
-        self.version = "0.0.0.0" if version is None else version
+        self.version = VERSION_0 if version is None else version
         self.uris = []
 
+    @property
+    def requested_version(self):
```

Comment:
I think having a requested_version and a version properties is confusing... maybe the latter should be called requested_version_string?